ivy/container/wrapping.py

```diff
@@ -54,7 +57,8 @@ def map_fn(vals, _):
             kw = ivy.copy_nest(kwargs, to_mutable=True)
             ivy.set_nest_at_indices(kw, kwarg_cont_idxs, kwarg_vals)
             return fn(*a, **kw)
-
+        # Call the container version of the function on every container in args and 
+        # kwargs and store the result in ret
```

Comment:
Not quite, we call the normal ivy version of the function `fn` on each array at the leaves of the containers. `map_fn` replaces each container arg and kwarg with the array at this specific leaf, before passing to the standard ivy method `fn`